parse/TechsParser.cpp

```diff
@@ -105,8 +105,19 @@ namespace {
         std::vector<std::shared_ptr<Effect::EffectsGroup>> effectsgroups;
         if (kw.has_key("effectsgroups")) {
             boost::python::stl_input_iterator<effect_group_wrapper> effectsgroups_begin(kw["effectsgroups"]), effectsgroups_end;
-            for (auto it = effectsgroups_begin; it != effectsgroups_end; ++it)
-                effectsgroups.push_back(it->effects_group);
+            for (auto it = effectsgroups_begin; it != effectsgroups_end; ++it) {
+                const auto& effects_group = *it->effects_group;
+                effectsgroups.push_back(std::make_unique<Effect::EffectsGroup>(
+                    std::move(ValueRef::CloneUnique(effects_group.Scope())),
```

Comment:
```suggestion
                    ValueRef::CloneUnique(effects_group.Scope()),
```
moving from what's already an rvalue is redundant or a pessimization, as it prevents return value optimization / move elision.

Reply:
Fixed.